test/types/string.js

```diff
@@ -561,7 +551,8 @@ describe('string', () => {
             Helper.validate(schema, [
                 ['joe@example.com', true],
                 ['joe@www.example.com', true],
-                ['joe@localhost', false, null, '"value" must be a valid email'],
```

Comment:
> I'd actually like to see the same test, but true
I'm not sure why the Github UI is showing that line just taken out. I didn't remove it, I did just [change the assertion](https://github.com/WesTyler/joi/blob/6af9763e13fd0fd18afd6b720afa790e0c8467c8/test/types/string.js#L565) to `true` :)